pkg/commands/fmt.go

```diff
@@ -69,6 +70,7 @@ func newFmtCommand(logger logutils.Log, info BuildInfo) *fmtCommand {
 	setupFormattersFlagSet(c.viper, fs)
 
 	fs.BoolVarP(&c.opts.diff, "diff", "d", false, color.GreenString("Display diffs instead of rewriting files"))
+	fs.BoolVar(&c.opts.stdin, "stdin", false, color.GreenString("Use standard input"))
```

Comment:
It's not clear from the description for what stdin is used. Maybe better is `Use standart input for piping source files`.

Inspired by https://cs.opensource.google/go/x/tools/+/refs/tags/v0.31.0:cmd/goimports/goimports.go;l=79